cli.go

```diff
@@ -1094,6 +1110,22 @@ func actionInspect(c *cli.Context) error {
 	if pub {
 		pubObjs(topub, cl, c)
 	}
+	if qr {
+		// Generate a QR code for each entity with an available signing key
+		for _, qrd := range(toqrg) {
+			roi := qrd.ro
+			if (roi.GetRONum() == objects.ROEntity) {
```

Comment:
I thought so too, but the function used to create Entities (https://github.com/immesys/bw2/blob/dev/objects/routing.go#L1607) checks if the RONum is ROEntityWKey, and if it is, sets it to ROEntity. So GetRONum() would still return ROEntity in this case. That's why I make sure that the RONum is ROEntity and make sure that the signing blob isn't nil.